analytics/custom_queries/templates/0101-2061.json

```diff
@@ -0,0 +1,20 @@
+{
+    "should": [
+        {
+            "title": "Food Science and Technology (Campinas)",
+            "fuzziness": 3
+        },
+        {
+            "title": "Food Sci. Technol (Campinas)",
+            "fuzziness": 3
+        },
+        {
+            "title": "Food Science and Technology",
+            "fuzziness": 0
+        },
+        {
+            "title": "Ciencia e Tecnologia dos Alimentos",
+            "fuzziness": 0
+        }
+    ]
+}
```

Comment:
Ok, adicionei outros tb. 
        "Wood Science and Technology",
        "Wood science and technology",
        "Wood Science and Technology,",
        "Wood Science and Technology.",
        "Wood Science and technology",
        "Wood science and Technology",
        "Wood. Science and Technology",
        "Wood Science and Tech nology",
        "Wood Science and Techology",
        "Wood Science and Tecnology"